### ISSUE TITLE
It runs projectile-ag instead of helm-projectile-ag

### ISSUE THREAD

**Issue by tsolar** [bbatsov/helm-projectile#30]
I have activated projectile and helm-projectile. I've updated to last MELPA version and now `C-c p s s` runs `projectile-ag` instead of `helm-projectile-ag`, same for `-grep` and others.

When I run `helm-M-x` it appears like this:
![captura de pantalla de 2016-06-07 16-15-51](https://cloud.githubusercontent.com/assets/1470901/15873140/8dfbadc2-2cd3-11e6-9535-e4dfd2b9ef09.png)

Do I have to remap something? How can I do that?

Thanks in advance!

**Comment by novakboskov** [comment]
I found the same with `helm-projectile-grep`.
Other functions are bound to keystrokes and work well (such as `helm-projectile-find-file`) despite that ` ... <remap> <projectile-find-file>`, except `helm-projectile-ack` which has no binding.

**Comment by svetlyak40wt** [comment]
That is because `remap` does not work. Guys decided to rollback it in issue #27

**Comment by angrybacon** [comment]
@tsolar As a temporary fix:

``` elisp
(use-package helm-projectile
  :after helm
  :bind
  (:map projectile-command-map
        ("b" . helm-projectile-switch-to-buffer)
        ("d" . helm-projectile-find-dir)
        ("f" . helm-projectile-find-file)
        ("p" . helm-projectile-switch-project)
        ("s s" . helm-projectile-ag))
  :config (helm-projectile-toggle 1))
```

**Comment by tsolar** [comment]
@angrybacon I could make it work with the fix. Thank you!

However, I think I should not have to patch/fix it. This should be working as before, "automagically".

**Comment by Bounga** [comment]
Latest update of helm / helm-core seems to fix the issue.

**Comment by jwintz** [comment]
I have every packages up to date and configured `helm-projectile` to bind `C-c p s s` to `helm-projectile-ag`, and still, the key combination runs `projectile-ag`. Using `M-x helm-projectile-ag` however works as expected.

```
(use-package helm-projectile
  :bind (
  ("C-c p h" . helm-projectile)
  ("C-c p a" . helm-projectile-find-other-file)
  ("C-c p f" . helm-projectile-find-file)
  ("C-c p F" . helm-projectile-find-file-in-known-projects)
  ("C-c p g" . helm-projectile-find-file-dwim)
  ("C-c p d" . helm-projectile-find-dir)
  ("C-c p e" . helm-projectile-recentf)
  ("C-c p b" . helm-projectile-switch-to-buffer)
  ("C-c p s g" . helm-projectile-grep)
  ("C-c p s a" . helm-projectile-ack)
  ("C-c p s s" . helm-projectile-ag)))
```

I'm running emacs 25.1.50.1

**Comment by fritzgrabo** [comment]
@jwintz, you're probably just missing a `(helm-projectile-toggle 1)`?

This works for me:
```
(use-package helm-projectile
  :config
  (helm-projectile-toggle 1))
```

**Comment by xiongtx** [comment]
This should be fixed by #31.
